### ISSUE TITLE
Concatenate js resource

### ISSUE THREAD

**Issue by jeanlaurent** [CodeStory/fluent-http#56]
When we have something like this

``` html
    <!-- concat:app.js-->
    <script src="/js/lib/jquery.min.js"></script>
    <script src="/js/lib/angular.min.js"></script>
    <script src="/js/lib/angular-route.min.js"></script>
    <script src="/js/lib/angularLocalStorage.js"></script>
    <script src="/js/lib/angular-cookies.min.js"></script>
    <script src="/js/lib/angular-animate.min.js"></script>
    <script src="/js/order.coffee"></script> <!-- only usefull for checkout -->
    <script src="/js/basket.coffee"></script>
    <script src="/js/webshop.coffee"></script>
   <!-- concat:end-->
```

we should concat in prod mode to 

``` html
    <script src="/js/app.js"></script>
```

this can also be done for css.

**Comment by dgageot** [comment]
The [[script]]  and [[css]] handlebars helpers support a list of names. It could concatenate the resources and link to a sha1

**Comment by BodySplash** [comment]
Would it be possible to support requirejs or browserify in addition or instead of this solution ?

**Comment by dgageot** [comment]
Is there something that prevents you from using browserify right now?

**Comment by BodySplash** [comment]
Not really. I can use browserify from the command line, and with a few scripts build my assets during continuous integration run. But it feels ankward to maintain several build scripts to package my application. I really don't know if it's possible, it's just an idea :) In my experience, js file concatenation is a less than ideal solution to package js assets  : 
* we can forget to add a script tag 
* in our js files, dependencies are never explicit, and we easily forget where a method or class comes from
* it encourages to use global variables
